.github/workflows/main.yml

```diff
@@ -47,6 +47,7 @@ jobs:
 
     - name: Install apt dependencies
       run: |
+        sudo apt update
```

Comment:
```suggestion
        sudo apt-get update
```

I don't think it makes any difference here. But apt-get is supposed to stay more common in scripting, I understand. Also this way it's using consistently the same (apt vs apt-get).